src/Microsoft.Framework.Configuration/ConfigurationBase.cs

```diff
@@ -69,16 +71,14 @@ public IList<IConfigurationSource> Sources
 
         public IEnumerable<IConfigurationSection> GetChildren()
         {
-            var prefix = GetPrefix();
-
             var segments = Sources.Aggregate(
                 Enumerable.Empty<string>(),
-                (seed, source) => source.ProduceConfigurationSections(seed, prefix, Constants.KeyDelimiter));
+                (seed, source) => source.ProduceConfigurationSections(seed, GetPath(), Constants.KeyDelimiter));
```

Comment:
It seemed fine to call this one and assign it to a variable in the beginning. It saves unnecessary method calls.